pkg/types/validation/installconfig.go

```diff
@@ -231,11 +231,14 @@ func validateNetworkingIPVersion(n *types.Networking, p *types.Platform) field.E
 			allErrs = append(allErrs, field.Invalid(field.NewPath("networking", "serviceNetwork"), strings.Join(ipnetworksToStrings(n.ServiceNetwork), ", "), "when installing dual-stack IPv4/IPv6 you must provide two service networks, one for each IP address type"))
 		}
 
+		allowV6Primary := false
```

Comment:
I would consider renaming variable to something like `allowDualStackV6Primary` to remove confusion with single-stack IPv6 configurations. We know today it's about dual-stack, but who knows what we are going to think about this variable in the future. Anyway your final call, for me lgtm anyway

Reply:
Eventually I'm hoping this variable can go away completely. Once we have upstream support for dual stack we should be able to allow v6-primary on any platform that supports dual stack.